drivers/Ticker.h

```diff
@@ -100,6 +100,10 @@ class Ticker : public TimerEvent, private NonCopyable<Ticker> {
      *
      *  @param func pointer to the function to be called
      *  @param t the time between calls in micro-seconds
+     *
+     *  @note setting t to very small value much less than RTOS tick frequency
```

Comment:
I would reference the t `@a t` also mention some numbers noting it's platform dependant.